Sources/PrimerSDK/Classes/Modules/UserInterfaceModule.swift

```diff
@@ -944,14 +944,28 @@ class UserInterfaceModule: NSObject, UserInterfaceModuleProtocol {
             PrimerPaymentMethodType.xfersPayNow.rawValue:
             let btn = PrimerButton()
             btn.isEnabled = false
+            btn.accessibilityIdentifier = "submit_btn"
```

Comment:
It's missing from `adyenMultibanco`, also maybe ut should be added on all buttons.